backend/core/forms.py

```diff
@@ -0,0 +1,11 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+
+
+class UserRegisterForm(UserCreationForm):
+    email = forms.EmailField()
+
+    class Meta:
+        model = User
+        fields = ["username", "email"]
```

Comment:
No password & confirmPassword fields ?

Reply:
I see it added here though https://github.com/Monstarrrr/rebutify/pull/31/commits/c9337e8c43aed4d818a62f649606ff648a5642a1 , I don't understand.

Reply:
I see you removed them here https://github.com/Monstarrrr/rebutify/pull/31/commits/2410fa92594e027b533db045a367cacbb0a26f4b

Reply:
those fields never existed in the creation form in the first place, so I removed them